src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -240,7 +240,7 @@
         )
     ),
     ParameterNameSegment=RegexParser(
-        r"[A-Z_][A-Z0-9_$]*", CodeSegment, name="parameter", type="parameter"
+        r'[A-Z_][A-Z0-9_$]*|"[^"]*"', CodeSegment, name="parameter", type="parameter"
```

Comment:
This regex doesn't appear to allow "escaping" quotes. Should it?

Reply:
I don’t think it should. The original implementation didn’t and this just allows quotes around it.

And really, what manic would use that in a parameter name! 😄